apps/io-wallet-user-func/src/infra/http/handlers/create-wallet-instance.ts

```diff
@@ -31,6 +44,29 @@ type WalletInstanceRequestPayload = t.TypeOf<
   typeof WalletInstanceRequestPayload
 >;
 
+// const getUserEmailByFiscalCode = (
+//   fiscalCode: string,
+// ): TE.TaskEither<Error, string> => TE.right(`${fiscalCode}@test.test`); // to do - [SIW-1560] get the user email by the fiscal code
+
+export const getTransporter: () => RTE.ReaderTaskEither<
+  { mail: MailConfig },
+  Error,
+  MailerTransporter
+> = () => (configs) =>
+  TE.tryCatch(
+    async () =>
+      await getMailerTransporter({
+        MAIL_FROM: configs.mail.mailSender,
+        MAIL_TRANSPORTS: undefined,
+        MAILHOG_HOSTNAME: undefined,
+        MAILUP_SECRET: configs.mail.mailupSecret,
+        MAILUP_USERNAME: configs.mail.mailupUsername,
+        NODE_ENV: "production",
+        SENDGRID_API_KEY: undefined,
```

Comment:
Why do you use sendgrid here?

Reply:
`getMailerTransport()` wants exactly this attribute set to `undefined` for MailUp. Otherwise, `getMaileTransport()` returns a misconfiguration error.

Reply:
Ok thanks, then leave a comment about it, maybe referencing the [type](https://github.com/pagopa/io-functions-commons/blob/1cf04d9ce5a96e7f7bbe8667cf8c818530c0b28c/src/mailer/config.ts#L37)